ipaserver/plugins/baseldap.py

```diff
@@ -715,7 +715,7 @@ def get_memberofindirect(self, entry):
             result = self.backend.get_entries(
                 self.api.env.basedn,
                 filter=filter,
-                attrs_list=[''])
+                attrs_list=[''], size_limit=0)
```

Comment:
A few lines above this the very same is done using `size_limit=-1, paged_search=True` so I'd rather that we stayed consistent.